napari/utils/events/evented_model.py

```diff
@@ -78,6 +88,18 @@ class EventedMetaclass(main.ModelMetaclass):
     """
 
     def __new__(mcs, name, bases, namespace, **kwargs):
+        # pydantic fails in a number of ways with annotated generics
```

Comment:
Do have a test or an example you could point to that shows these failures? I'm trying to reproduce what I interpret here, but cannot. I tried to use the tests added in this PR, but also found they worked after adding a simple pydantic validator.

Reply:
Ah, sorry, followed some other links that you posted and found this: https://github.com/samuelcolvin/pydantic/issues/1852